N28: Raytracer from Scratch (#857)
rust-gamedev/rust-gamedev.github.io: content/news/028/index.md

## Learning Material Updates

### [The Raytracer Challenge][raytracer-challenge-github]

![The Raytracer Challenge](the-raytracer-challenge.jpg)

[The Raytracer Challenge][raytracer-challenge-github] is a project with the goal

> to write a raytracer from scratch in rust, while showing each step of the way

Reviewer: ```suggestion
to write a raytracer from scratch in Rust, while showing each step of the way
```

[as a weekly live coding session][raytracer-challenge-playlist]. Everything is
documented, starting with implementing [Vectors][raytracer-challenge-vector] and
[Matrices][raytracer-challenge-matrix] all the way to creating [Phong
Lighting][raytracer-challenge-phong].

> November has been a busy month for the project with a lot visual changes in the

Reviewer: ```suggestion
November has been a busy month for the project with lots of visual changes in the
```

raytraced results:

- A [camera][raytracer-challenge-camera] was added to allow arbitrary views on
  the [World][raytracer-challenge-world].

- [Basic animation support][raytracer-challenge-animation] came along allowing
  the easy creation of video sequences.

> - More realistic and life like scenes due to [Shadow

Reviewer: ```suggestion
- More realistic and life-like scenes due to [Shadow
```
